### PR TITLE
Add-batch-cert-script

### PR DESCRIPTION
**Pull request by edcdavid**
Moving improved operator batch script in cnf repo. Added support for html page

### CODE REVIEW

**Comment by rdavid** [comment]
- Put the script to dir `script`.
- Use tab instead of spaces for shfmt linter.

File: run-basic-batch-operators-test.sh [hunk 1]
```diff
@@ -0,0 +1,208 @@
+#!/bin/sh
+
+# Test run timestamp
+TIMESTAMP=$(date | sed 's/[ :]/_/g')
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:4]: The timestamp format is not really timestamp.
```suggestion
TIMESTAMP=$(date +%Y%m%d-%H:%M:%S)
```

File: run-basic-batch-operators-test.sh [hunk 2]
```diff
@@ -0,0 +1,208 @@
+#!/bin/sh
+
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:2]:
```suggestion
set -o errexit -o nounset -o pipefail

```

File: run-basic-batch-operators-test.sh [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+# Test run timestamp
+TIMESTAMP=$(date | sed 's/[ :]/_/g')
+
+# base folder
+BASE_DIR=/var/www/html
+
+# INPUTS
+
+# config.yaml template file path
+CONFIG_YAML_TEMPLATE=$(pwd)/tnf_config.yml.template
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:12]:
```suggestion
CONFIG_YAML_TEMPLATE="$(pwd)"/tnf_config.yml.template
```

File: run-basic-batch-operators-test.sh [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+# Location of telco/non-telco classification file
+CNF_TYPE=cmd/tnf/claim/show/csv/cnf-type.json
+
+# operator bundle list from user
+ORIG_BUNDLE_PATH=$1
+
+# OUTPUTS
+
+# report folder
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:25]: Start all comments with a capital letter.

File: run-basic-batch-operators-test.sh [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+CONFIG_YAML_TEMPLATE=$(pwd)/tnf_config.yml.template
+
+# Docker config used to pull operator images
+DOCKER_CONFIG=config.json
+
+# Location of telco/non-telco classification file
+CNF_TYPE=cmd/tnf/claim/show/csv/cnf-type.json
+
+# operator bundle list from user
+ORIG_BUNDLE_PATH=$1
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:21]: It should be assigned after checking if the number of parameters is correct.

File: run-basic-batch-operators-test.sh [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+	# store the results of CNF test in a new directory
+	mkdir -p "$reportDir"
+
+	configYaml="$reportDir"/tnf_config.yml
+
+	# change the targetNameSpace in tng_config file
+	sed "s/\$ns/$ns/" "$CONFIG_YAML_TEMPLATE" >"$configYaml"
+
+	# label everything
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:156]: It is not obvious which labels are added. Provide examples in the comment.

**Review by sebrandon1 — APPROVED** [review]

File: run-basic-batch-operators-test.sh (hunk 2 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh:3]: fixed

File: run-basic-batch-operators-test.sh (hunk 1 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

File: run-basic-batch-operators-test.sh (hunk 3 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

File: run-basic-batch-operators-test.sh (hunk 4 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: Fixed

File: run-basic-batch-operators-test.sh (hunk 5 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

File: run-basic-batch-operators-test.sh (hunk 6 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

**Pull request merged**